scripts/ci/build_fusion_and_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+#
+# Copyright 2017 Open GEE Contributors
+#
+# Licensed under the Apache license, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+# This script is used by Travis build stages (see travis.yml at root) that build
+# fuison and run unit tests. Assumes the following environment variables are set:
+# CPP_STD    - Should be set to `98` or `11` depending on the standard to build with
+# BUILD_TYPE - The type of build `release`, `optimize`, or `internal`
+
+case "$CPP_STD" in
+    98|11)
```

Comment:
we may add 14 or 17 one day...this does make it clean and simple to specify which version(s) we will build with.

Reply:
Yes we can just update this as we add/remove support for different standards.